Selection_Sort/Selection_Sort.ts

```diff
@@ -0,0 +1,31 @@
+// function for selection sort logic
+function Selection_Sort(numbers: number[]) : number[]{
+
+    let i: number;
+    let j: number;
+    let min: number;
+    let temp: number;
+
+    for(i = 0; i < numbers.length - 1; i++){
+        min = i;
+        for(j = i + 1; j < numbers.length; j++){
+            if(numbers[j] < numbers[min]){
+                min = j;
+            }
+        }
+        if(i !== min){
+            temp = numbers[i];
+            numbers[i] = numbers[min];
+            numbers[min] = temp;
+        }
+    }
+
+    return numbers;
+}
+
+// array of numbers (INPUT)
+const numbers: number[] = [10, 9, 8, 1, 5, 2, 4, 7, 3];
+
+// OUTPUT ([ 1, 2, 3, 4, 5, 7, 8, 9, 10 ])
+console.log(Selection_Sort(numbers));
+
```

Comment:
sample input/output should be here

Reply:
@raghav-dalmia I have mentioned the sample input/output already. Do you want me to add more examples?

Reply:
I am saying, it should be at the bottom of the file.

Reply:
@raghav-dalmia I got it. Will do it in a while and let you know.